packages/mdctl-cli/tasks/study.js

```diff
@@ -238,7 +238,8 @@ class Study extends Task {
     Arguments:               
       
       Command                        
-        export - Exports the study from the current org      
+        export - Exports the study from the current org
+        import - Imports the study from the current org
```

Comment:
I'd change the grammar here a little bit, I think  import explanation would read better if it was: "Imports the study into the current org"